core/src/main/kotlin/io/specmatic/core/pattern/JSONArrayPattern.kt

```diff
@@ -13,6 +13,11 @@ import io.specmatic.core.value.Value
 import java.util.*
 
 data class JSONArrayPattern(override val pattern: List<Pattern> = emptyList(), override val typeAlias: String? = null) : Pattern, SequenceType {
+
+    override fun eliminateOptionalKey(value: Value, resolver: Resolver): Value {
+        return JSONArrayValue(emptyList())
```

Comment:
Either we should fully implement, or not at all. Given this is a legacy pattern, I suggest not implementing for now.

Reply:
Thanks, removed the implementation